lib/performance.ex

```diff
@@ -0,0 +1,88 @@
+defmodule Performance do
+  # Function to run tests with different parameter combinations
+  def run_tests(test_params_list) do
+    Enum.map(test_params_list, &start_test/1)
+  end
+
+  # Function to start the test with given parameters
+  def start_test(%{num_crdts: num_crdts, num_processes_per_crdt: num_processes_per_crdt, num_iterations_per_process: num_iterations_per_process}) do
+    # Start the timer for the whole test
+    start_test_time = System.monotonic_time()
+
+    # Start CRDTs
+    crdts = start_crdts(num_crdts)
```

Comment:
Starting them explicitly won't work because our entire ecosystem must be in `LCRDT.Network.all_nodes`. Actually there's no way we can easily start them like this. I think we will just have to change the code for all our trials :(